google/ggrc-core
# GGRC-3495/4041 Update assessment logic to accept attachments and/ or urls as evidence

**Note:** Contains [GGRC-4041](https://github.com/google/ggrc-core/pull/7075)

# Issue description

Currently user can create an assessment template with a local custom attribute of type dropdown with a list of options. For each option he can mark that it is required to attach file or/and add comment, if option is selected.

Now user would like to mark 'evidence urls' as required for such options.

# Steps to test the changes

User generates an assessment.
User adds an evidence url
User select an option in local custom attribute (type = dropdown), that requires url.
System shouldn't ask to attach urls, answer is valid.
User select an option in another local custom attribute (type = dropdown), that requires url.
System should ask to attach url.

# Requirements 

See ticket GGRC-3495 for acceptance criteria and details.

# Solution description

See commits.

# Sanity checklist




# PR Review checklist
